### DISCUSSION TITLE
Marker Styles

### DISCUSSION THREAD

**Discussion by jllucero** [opengeos/leafmap#53] [Q&A] [answered]
I'd like to change the marker styles.  I have many points that I need to plot in California.  But the default markers are too large such that the entire map is covered in markers.  Can someone show me how to change the styles, size, color, etc.?

Thank you!

**Comment by giswqs** [comment, marked as answer]
Try Market Cluster. Also see ipyleaflet [AwesomeIcon](https://ipyleaflet.readthedocs.io/en/latest/api_reference/awesome_icon.html). 

![](https://i.imgur.com/hnaTPZa.gif)

**Reply by jllucero** [reply]
Thank you!

**Reply by jllucero** [reply]
I'm still having trouble changing the markers to circle markers.  When I try to import using the following code I get an error

from leafmap.foliumap import CircleMarker, Marker

What I would like are simple dots or points for each of my locations.  I'd appreciate any suggestions.
Thank you.

**Reply by giswqs** [reply]
You should use `from folium import CircleMarker, Marker`

**Reply by jllucero** [reply]
Thank you!  It works great!
